uint8 node_id
---
bool success